web/src/components/MapView.tsx

```diff
@@ -88,22 +92,25 @@ const MapView: React.FunctionComponent<MapViewProps> = (props: MapViewProps): Re
       <ReactMapGL
         interactiveLayerIds={[layerStyle.id!]}
         {...viewport}
+        height='100%'
+        width='100%'
         onViewportChange={setViewport}
         mapStyle={mapConfig.styleJSON}
-        onClick={clickItem}>
+        onClick={clickItem}
+        // @ts-ignore BottomSheetRef does not provide current
+        onTouchMove={() => ref?.current.snapTo(({ maxHeight }: { maxHeight: number }) => getSnapPoints(maxHeight)[0])}>
         {/* To use geolocation in a development build you have to start the dev server with "yarn start --https" */}
         <GeolocateControl
           style={geolocateControlStyle}
           positionOptions={{ enableHighAccuracy: true }}
           trackUserLocation
         />
-        <Source id='location-pois' type='geojson' data={featureCollection}>
+        <Source id='location-pois' type='geojson' data={featureCollection!}>
```

Comment:
if `featureCollection` is null here this will render an error.
```suggestion
        <Source id='location-pois' type='geojson' data={featureCollection}>
```

Reply:
resolved this be a null check!